cluster/gce/gci/configure-helper.sh

```diff
@@ -1715,6 +1731,67 @@ function start-kube-apiserver {
   sed -i -e "s@{{admission_controller_config_volume}}@${admission_controller_config_volume}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_mount}}@${image_policy_webhook_config_mount}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_volume}}@${image_policy_webhook_config_volume}@g" "${src_file}"
+
+  if [[ -z ${ETCD_KMS_KEY_ID:-} ]]; then
+    # Removing KMS related placeholders.
+    sed -i -e " {
```

Comment:
how does this multiline sed program work? does it execute all of these edits in one invocation?